odoo/addons/base/models/ir_asset.py

```diff
@@ -359,7 +359,7 @@ def is_safe_path(path):
             # files are read from the file system. But web assets (scripts and
             # stylesheets) must be loaded using relative paths, hence the trimming
             # for non-xml file paths.
-            paths = [path if path.split('.')[-1] in TEMPLATE_EXTENSIONS else path[len(addons_path):] for path in paths]
+            paths = [path if path.split('.')[-1] in TEMPLATE_EXTENSIONS else fs2web(path[len(addons_path):]) for path in paths]
```

Comment:
Shouldn't the path be fs2web'd in both cases here?

Reply:
Template files are read from the file system, so their path should not be fs2web´d